libs/ledger-live-common/src/csvExport.ts

```diff
@@ -65,7 +67,10 @@ const fields: Field[] = [
   {
     title: "Account Name",
     // FIXME: we need to inject wallet state if we want the actual user's account name
-    cell: (account, parentAccount) => getDefaultAccountName(getMainAccount(account, parentAccount)),
+    cell: (account, parentAccount, _op, _counterValueCurrency, _countervalueState, walletState) =>
```

Comment:
why do we need this 3 ? `_op, _counterValueCurrency, _countervalueState`

Reply:
Because how it's done today each param if walletState is at the third place it would be the operation

Reply:
indeed ^^